crates/ruma-html/src/sanitizer_config/clean.rs

```diff
@@ -30,8 +30,14 @@ static ALLOWED_ATTRIBUTES_STRICT: Map<&str, &Set<&str>> = phf_map! {
     "code" => &ALLOWED_ATTRIBUTES_CODE_STRICT,
     "div" => &ALLOWED_ATTRIBUTES_DIV_STRICT,
 };
+
+#[cfg(not(feature = "unstable-msc4286"))]
```

Comment:
This is not ideal because it will become complicated if there are more attributes to stop ignoring… But I guess it's good for now as the `phf_set!` macro doesn't allow to add entries behind a feature flag, so otherwise it would probably require a breaking change.

Reply:
Agreed. I originally tried to achieve something like this:

```rust
static ALLOWED_ATTRIBUTES_SPAN_STRICT: Set<&str> =
    phf_set! { "data-mx-bg-color", "data-mx-color", "data-mx-spoiler", "data-mx-maths",
    #[cfg(feature = "unstable-msc4286")]
    "data-msc4286-external-payment-details-maths",
    };
```

But, as you say, the macro doesn't allow it.